www/js/components/BarChart.tsx

```diff
@@ -20,13 +20,13 @@ Chart.register(
 );
 
 const BarChart = ({ chartData, axisTitle, lineAnnotations=null, isHorizontal=false }) => {
-
+ 
   const { colors } = useTheme();
   const [ numVisibleDatasets, setNumVisibleDatasets ] = useState(1);
 
   const barChartRef = useRef<Chart>(null);
 
-  const defaultPalette = [
+  const defaultPalette = [ 
```

Comment:
Try to avoid committing needless changes like this, where you add a space or adjust indentation

I know this was probably on accident - just make sure you check over the diff of what you're committing before you do so